app/css/lab-orders-list.scss

```diff
@@ -46,3 +54,84 @@
     max-width: 90px;
 }
 
+.name-emrid-order-filter {
+  display: flex;
+  width: 75%;
+  position: relative;
+
+  input {
+    width: 100% !important;
+    padding: 0px 0px 0px 40px !important;
+    border-radius: 0px !important;
+    font-size: 16px;
+    background-color: #FFF;
+    border: 1px solid #DDD;
+    color: #003f5e;
+    display: block;
+    height: 45px;
+  }
+  .icon-search{
+    position: absolute;
+    left: 1%;
+    line-height: 40px;
+  }
+  .icon-remove-sign {
+    position: absolute;
+    left: 94%;
+    line-height: 40px;
+  }
+}
+
+.order-list-filters {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  position: relative;
+  top: 100px;
+  .bottom-filters {
+    width: 100%;
+    display: flex;
+    .form-filter-group {
```

Comment:
As discussed let's move all shared components styles to their specific css/scss files